### DISCUSSION TITLE
Merge Go-Ahead / Arverio trips

### DISCUSSION THREAD

**Discussion by jheubuch** [Traewelling/traewelling#3032] [General]
After the rebranding from Go-Ahead Deutschland to Arverio Deutschland, the HAFAS Operator IDs changed several times.

My Idea would be to either 
- merge all past Go-Ahead data to the new Arverio operator 
or 
- keep the Go-Ahead data before the rebranding at 10.07.2024 and merge all the following trips into the current operator ID.

**Comment by jheubuch** [comment]
For Go-Ahead Bayern I have those operator data:

- `go-ahead-bayern-gmbh` as `Go-Ahead Bayern GmbH` (data before rebranding)
- `goahead-bayern-arverio-bayern` as `GoAhead Bayern  - Arverio Bayern` (rebranding phase)
- `goahead-arverio-bayern` as `GoAhead - Arverio Bayern` (rebranding phase)
- `arverio-bayern` as `Arverio Bayern` (after rebranding)

I think for Baden-Württemberg those all do also exist.

**Reply by MrKrisKrisu** [reply]
Ich wäre hier glaube etwas pingelig und würde nicht alles umbenennen. `go-ahead-bayern-gmbh` war damals ja korrekt, daher würde ich vorschlagen das nicht anzurühren. Man ist ja mit Go-Ahead gefahren.

Die zwei Werte während der „rebranding phase” würde ich aber zu `arverio-bayern` mergen. Würde das auf Zustimmung stoßen?

**Reply by jheubuch** [reply]
Find ich gut. :)

**Reply by MrKrisKrisu** [reply]
✅

- 135 Trips: `goahead-bayern-arverio-bayern` -> `arverio-bayern`
- 461 Trips: `goahead-arverio-bayern` -> `arverio-bayern`
